### PR TITLE
Added GeoZone option to create, update, copy, and restore workflows

### PR DESCRIPTION
**Pull request by rebeccaxu-ms**
## Description

Adding the GeoZone storage account type option to the following commands:
New-AzSqlDatabase
Set-AzSqlDatabase
New-AzSqlDatabaseCopy
New-AzSqlDatabaseSecondary
Restore-AzSqlDatabase

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/Sql/Sql/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {[PR link here](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1186)}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by BethanyZhou — COMMENTED** [review]
Generally looks good to me. Please add change log and update cmdlets' help markdown.

**Comment by wyunchi-ms** [comment]
Ping @rebeccaxu-ms

File: src/Sql/Sql/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+* Added `GeoZone` option to `BackupStorageRedundancy` parameter to `New-AzSqlDatabase`, `Set-AzSqlDatabase`, `New-AzSqlDatabaseCopy`, `New-AzSqlDatabaseSecondary`, and `Restore-AzSqlDatabase` to enable RAGZRS create, update, copy, geo secondary and PITR support for hyperscale databases
```
Comment (pixia) [inline src/Sql/Sql/ChangeLog.md:22]: change to "to enable create, update, copy, geo secondary and PITR support for GEOZONE hyperscale databases"

**Review by pixia — APPROVED** [review]

**Comment by wyunchi-ms** [comment]
Please check in the recording files of the test cases.

**Pull request merged**